src/App.vue

```diff
@@ -16,111 +16,102 @@ export default {
 		AppNavbar,
 		AppFooter
 	},
+	data() {
+		return {
+			dark: JSON.parse(localStorage.getItem('dark'))
```

Comment:
this should be computed and not in data

Reply:
this can't be computed, as we need to actually change state for the site to change it's look.